docs/CONTRIBUTING.md

```diff
@@ -20,10 +20,10 @@ with both to a certain extent. If not, the following references might be useful:
 
 ## First steps
 
-DISCO is a complex project composed of the Disco.js library (`discojs`), a front-end (`web-client`),
-a `server` and a `cli` (e.g., for benchmarking). Depending on what your goal is, you might only use a subset of them, e.g. you won't need an in-depth understanding of the web-client and Vue.js to add a new decentralized learning feature. Instead, you will probably rely on the CLI.
+DISCO is a complex project composed of the Disco.js library (`discojs`), a front-end (`webapp`),
```

Comment:
By renaming `discojs-core`into `discojs`we don't really have a name to refer to the three discojs modules anymore, maybe we change reword to:
```suggestion
DISCO is a complex project composed of the Disco.js library (`discojs`, `discojs-node` and `discojs-node`), a front-end (`webapp`),
```